specifyweb/frontend/js_src/lib/components/FormParse/webOnlyViews.ts

```diff
@@ -84,6 +84,37 @@ export const webOnlyViews = f.store(() =>
       'edit',
       ['name']
     ),
+    [adminUser]: autoGenerateViewDefinition(
+      tables.SpecifyUser,
+      'form',
+      'edit',
+      ['name', 'password', 'agents']
```

Comment:
Field captions are usually determined by the schema configuration, but in the case the user sees this upon a first run, the captions will just be verbatim the name (e.g. "IsAccessionsGlobal" rather than a user-assigned schema). How are we going to set the labels?